bokeh/models/sources.py

```diff
@@ -38,6 +39,10 @@ class DataSource(PlotObject):
                 hit/selected
     """)
 
+    callback = Instance(Callback, help="""
+    A callback to run in the browser whenever the selection is changed.
+    """)
+
```

Comment:
I like the single callback with multiple kinds of events option better too..

btw, agree this is not for this (0.9 :wink: ) PR